build.gradle

```diff
@@ -61,25 +61,39 @@ minecraft {
 sourceSets.main.resources { srcDir 'src/generated/resources' }
 
 repositories {
-    flatDir {
-        dirs 'libs'
+    maven {
+        url "https://cursemaven.com"
+        content {
+            includeGroup "curse.maven"
+        }
     }
+
+    // flatDir {
+    //     dirs 'libs'
+    // }
 }
 
 dependencies {
     minecraft "net.minecraftforge:forge:${minecraft_version}-${forge_version}"
 
-    compileOnly fg.deobf("_:rubidium:0.5.6")
-//    implementation fg.deobf("_:rubidium:0.5.6")
+    compileOnly "curse.maven:$rubidium_project_id:$rubidium_file_id"
+    // compileOnly fg.deobf("_:rubidium:0.5.6")
+    // implementation fg.deobf("_:rubidium:0.5.6")
+
+    // implementation fg.deobf("_:Patchouli:1.18.2-71.1")
+    // implementation fg.deobf("_:TerraFirmaCraft:Forge-1.18.2-2.1.11-beta")
+    // implementation fg.deobf("_:Cartography:1.18.2-0.4.0-alpha3")
+
+    implementation fg.deobf("curse.maven:$architectury_project_id:$architectury_file_id")
+    implementation fg.deobf("curse.maven:$ftbchunks_project_id:$ftbchunks_file_id")
+    implementation fg.deobf("curse.maven:$ftblib_project_id:$ftblib_file_id")
+    implementation fg.deobf("curse.maven:$ftbteams_project_id:$ftbteams_file_id")
 
-//    implementation fg.deobf("_:Patchouli:1.18.2-71.1")
-//    implementation fg.deobf("_:TerraFirmaCraft:Forge-1.18.2-2.1.11-beta")
-//    implementation fg.deobf("_:Cartography:1.18.2-0.4.0-alpha3")
 
-    implementation fg.deobf("_:ftb-chunks:${ftbchunks_version}")
-    implementation fg.deobf("_:ftb-teams:${ftbteams_version}")
-    implementation fg.deobf("_:ftb-library:$ftblib_version")
-    implementation fg.deobf("_:architectury:${architectury_version}")
+    // implementation fg.deobf("_:ftb-chunks:${ftbchunks_version}")
+    // implementation fg.deobf("_:ftb-teams:${ftbteams_version}")
+    // implementation fg.deobf("_:ftb-library:$ftblib_version")
+    // implementation fg.deobf("_:architectury:${architectury_version}")
```

Comment:
these commented out lines need to go as well, since they now reference variables that don't exist and with curse maven these are also artifacts that cannot be found